contents/handbook/growth/marketing/value-propositions.md

```diff
@@ -14,30 +14,12 @@ PostHog has 3 value propositions, outlined below in order of importance. Each is
 
 ## 1. An all-in-one platform
```

Comment:
```suggestion
## An all-in-one platform for product engineers
```

Reply:
Ha, I deliberately wanted it to be a 1 item list! I suppose that is editorially not acceptable?

Reply:
Oh, I dunno. Seems a bit too subtle to not confuse people, imo.